elements/pfe-sass/mixins/_mixins.scss

```
@@ -1,23 +1,25 @@
 @mixin browser-query($browser-list) {
   @each $browser in $browser-list {
-
     @if $browser == ie11 {
       @media screen and (-ms-high-contrast: active), screen and (-ms-high-contrast: none) { /* IE10+ */
         @content;
       }
     }
-
     @if $browser == edge12 {
       @supports (-ms-accelerator: true) { /* Microsoft Edge Browser 12+ */
         @content;
+      }
     }
-  }
-
     @if $browser == edge {
       @supports (-ms-ime-align: auto) { /* Microsoft Edge Browser 16+ (All) */
         @content;
       }
     }
+    @if $browser==firefox {
```

```suggestion
    @if $browser == firefox {
```

To match formatting of other items in the list